device-types/Digi/Digi-CM-16.yml

```diff
@@ -0,0 +1,54 @@
+---
+manufacturer: Digi
+model: Digi CM 16
+slug: digi-cm-16
+part_number: '70001910'
+u_height: 1
+is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (5.80 lbs or 2.60 kg)